src/app/handlers/lionping.handler.ts

```diff
@@ -13,23 +12,17 @@ export class LionPingHandler implements IHandler {
     }
 
     if (!this._reactEmoji) {
-      this._reactEmoji =
-        message.guild?.emojis.cache.filter((e) => e.name === 'lion').first() || '👍';
+      this._reactEmoji = message.guild?.emojis.cache
+        .filter((e) => e.name === 'lion')
+        .first() ?? '👍';
```

Comment:
What is `||` vs `??`

Reply:
`??` Is the null coalescing operator, it's preferred over using || for checking for undefined or null, because || can give false positives for anything that evaluates to false. For example if an empty string is returned `||` will think the left hand side is false. `??` Would evaluate to true, which is the expected behavior because `""` isn't null or undefined.